lang/cs/Org.Apache.REEF.Tests/Functional/Failure/EvaluatorWithRunningTaskDelayedPoisonTest.cs

```diff
@@ -138,49 +125,14 @@ public void OnNext(IFailedEvaluator value)
                 {
                     throw new Exception("Collection of failed Contexts contains " + value.FailedContexts.Count + " failed Contexts but only " + expectedStr);
                 }
-                
+
                 if (!value.FailedContexts.Select(ctx => ctx.Id).Contains(ContextId))
```

Comment:
When the code reaches here, it guarantees the number of context is 1. Do you sill need to uses select and contains?